README.md

```diff
@@ -1,2 +1,32 @@
-# cyberwatch_api_toolbox
+# Cyberwatch API toolbox
 Some script to use Cyberwatch Api - www.cyberwatch.fr
```

Comment:
Remove this line

Reply:
Remove `Some script to use Cyberwatch Api - www.cyberwatch.fr` and add `# Cyberwatch Api Toolbox`